docs/user-guide.md

```diff
@@ -0,0 +1,31 @@
+
+In QGIS toolbars click on the STAC APIs Browser icon
+
+![image](images/toolbar.png)
+
+
+## Add a STAC Catalog
+
+**STAC Catalog** is a simple, flexible JSON file of links that provides a structure to organize and browse STAC Items
+
+![image](images/add-connection.png)
+
+## Search and add Item into QGIS
+
+### Using the search filters
+
+Searching STAC Item can be filtered by:
+
+* Collections
```

Comment:
We need to provide explanation of each search filter, about their respective usage in the plugin